src/ocamlformat_reason.ml

```diff
@@ -95,3 +95,24 @@ match Conf.action with
         to_output_file output_file s ;
         Caml.exit 0
     | Error _ -> Caml.exit 1 )
+| Check inputs ->
```

Comment:
Similar to `--inplace`, `Check` doesn't make sense for ocamlformat_reason - the source is a binary file